### PR TITLE
test: Run TestStorageLuks.testClevis also on the Debians

### PR DESCRIPTION
**Pull request by mvollmer**
- [x] #15844

### CODE REVIEW

**Comment by mvollmer** [comment]
This needs the bash->sh fixes from #15823, I'll move it over to here, as suggested.

File: pkg/storaged/clevis-luks-passphrase.sh [hunk 1]
```diff
@@ -21,7 +21,7 @@ if cryptsetup isLuks --type luks1 "$DEV"; then
     luksmeta show -d "$DEV" | while read slot state uuid; do
         if [ "$state" == "active" -a "$uuid" == "$CLEVIS_UUID" ]; then
             if pp=$(luksmeta load -d "$DEV" -s "$slot" | clevis decrypt); then
-                echo $pp
+                printf '%s\n' "$pp"
```
Comment (martinpitt) [inline pkg/storaged/clevis-luks-passphrase.sh:24]: I think you should then also change the hashbang to /bin/sh to indicate that. There are several test failures anyway, so you can slip that into the next push. Thanks!

Comment (mvollmer) [reply pkg/storaged/clevis-luks-passphrase.sh:24]: Ah, good point.

Comment (mvollmer) [reply pkg/storaged/clevis-luks-passphrase.sh:24]: Done, and fixed the rest as well.

**Review by martinpitt — DISMISSED** [review]
Hah, more bashisms. LGTM, good to go once tests go green. Danke!

**Comment by mvollmer** [comment]
ubuntu-2004 seems to have a old version of tang that still suffers from startup/key creation races.  We had workarounds for those for some time but removed them when the issues got fixed in the Fedoras.

Instead of bringing back the workarounds, let's avoid the races by not creating our own keys: #15844

**Review by martinpitt — APPROVED** [review]
All Debian/Ubuntu tests passed, yay!

File: test/verify/check-storage-luks [hunk 2]
```diff
@@ -139,11 +139,14 @@ class TestStorageLuks(StorageCase):
...
         wait(lambda: m.execute("(loginctl list-users | grep admin) || true") == "")
 
-    @skipImage("No clevis/tang", "debian-stable", "debian-testing", "ubuntu-2004", "ubuntu-stable")
     def testClevisTang(self):
         m = self.machine
         b = self.browser
 
+        if m.image == "ubuntu-stable":
+            # ubuntu-stable needs this workaround.
+            m.execute("mkdir -p /var/db/tang")
```
[the remark this answers is not in the document]
Comment (mvollmer) [reply test/verify/check-storage-luks:148]: Not that I know of but tang-8-2 in debian-testing doesn't have this problem while tang-8-1 in ubuntu-stable does. So I am guessing the bug is known and fixed and the fix will come to ubuntu eventually...

[the remark this answers is not in the document]
Comment (martinpitt) [reply test/verify/check-storage-luks:148]: Thanks! Indeed the [changelog for 8-2](https://tracker.debian.org/news/1239254/accepted-tang-8-2-source-into-unstable/) mentions exactly this.

**Pull request merged**